src/scripts/main.js

```diff
@@ -22,7 +22,21 @@ const food = {
 const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
-  // WRITE YOUR CODE HERE
+  const list = document.createElement('ul');
+
+  for (const key in data) {
+    const item = document.createElement('li');
+
+    item.textContent = key;
+
+    list.append(item);
+
+    if (Object.keys(data[key]).length !== 0) {
```

Comment:
I think, you can just
```suggestion
    if (Object.keys(data[key]).length) {
```